### PR TITLE
[chore] Add diffusers-format example to LongCatAudioDiTPipeline

### PR DESCRIPTION
**Pull request by RuixiangMa**
# What does this PR do?

* add diffusers-format example(repo_id: ruixiangma/LongCat-AudioDiT-1B-Diffusers)
* support seed parameter

```
import soundfile as sf
import torch
from diffusers import LongCatAudioDiTPipeline

pipeline = LongCatAudioDiTPipeline.from_pretrained(
    "ruixiangma/LongCat-AudioDiT-1B-Diffusers",
    torch_dtype=torch.bfloat16,
)
pipeline = pipeline.to("cuda")

prompt = "A calm ocean wave ambience with soft wind in the background."
audio = pipeline(
    prompt,
    audio_duration_s=5.0,
    num_inference_steps=20,
    guidance_scale=4.0,
    generator=torch.Generator("cuda").manual_seed(42),
).audios[0, 0]

sf.write("output.wav", audio, pipeline.sample_rate)
```
## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [ ] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by RuixiangMa** [comment]
@dg845  I uploaded a Diffusers-format repository, updated usage docs.

File: docs/source/en/api/pipelines/longcat_audio_dit.md [hunk 1]
```diff
@@ -31,27 +26,28 @@ import torch
...
 audio = pipeline(
-    prompt="A calm ocean wave ambience with soft wind in the background.",
-    audio_end_in_s=5.0,
+    prompt,
+    audio_duration_s=5.0,
     num_inference_steps=16,
     guidance_scale=4.0,
-    output_type="pt",
-).audios
+    seed=42,
```
Comment (dg845) [inline docs/source/en/api/pipelines/longcat_audio_dit.md]:
```suggestion
    generator=torch.Generator("cuda").manual_seed(42),
```
`diffusers` pipelines generally only support a `generator` argument for controlling randomness.

File: src/diffusers/pipelines/longcat_audio_dit/pipeline_longcat_audio_dit.py [hunk 2]
```diff
@@ -202,6 +220,7 @@ def __call__(
         latents: torch.Tensor | None = None,
         num_inference_steps: int = 16,
         guidance_scale: float = 4.0,
+        seed: int | None = None,
```
Comment (dg845) [inline src/diffusers/pipelines/longcat_audio_dit/pipeline_longcat_audio_dit.py]:
```suggestion
```
Following https://github.com/huggingface/diffusers/pull/13483#discussion_r3090145607, since `diffusers` pipelines usually only support a `generator` argument, I think it would be more clear to remove the `seed` argument in favor of using the `generator` argument.

File: docs/source/en/api/pipelines/longcat_audio_dit.md [hunk 3]
```diff
@@ -31,27 +26,28 @@ import torch
...
+sf.write("longcat.wav", audio, pipeline.sample_rate)
 ```
 
 ## Tips
 
-- `audio_end_in_s` is the most direct way to control output duration.
-- `output_type="pt"` returns a PyTorch tensor shaped `(batch, channels, samples)`.
+- `audio_duration_s` is the most direct way to control output duration.
+- `seed` makes generation reproducible (optional, defaults to None).
+- Output shape is `(batch, channels, samples)` - use `.audios[0, 0]` to get a single audio sample.
```
Comment (dg845) [inline docs/source/en/api/pipelines/longcat_audio_dit.md:50]: nit: I think it might be more clear here if we clarify how the pipeline handles mono and stereo outputs.

**Comment by dg845** [comment]
@bot /style

**Review by dg845 — APPROVED** [review]
Thanks for the follow-up PR! Left a few small comments/suggestions :).

File: docs/source/en/api/pipelines/longcat_audio_dit.md (hunk 3 above)
Comment (RuixiangMa) [reply docs/source/en/api/pipelines/longcat_audio_dit.md:50]: Added mono/stereo clarification in Tips

**Comment by RuixiangMa** [comment]
> Thanks for the follow-up PR! Left a few small comments/suggestions :).

Fixed, PTAL

**Comment by dg845** [comment]
@bot /style

**Review by dg845 — APPROVED** [review]
Thanks! (BTW, you can fix the code style with `make style` and `make quality`.)

**Comment by dg845** [comment]
Merging as the CI failures are unrelated.
